traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -534,7 +534,8 @@ <h3>Current Value</h3>
                         </div>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
-                            <textarea id="remapText" name="remapText" class="form-control" ng-model="deliveryService.remapText" rows="3"></textarea>
+                            <textarea id="remapText" name="remapText" class="form-control" ng-model="deliveryService.remapText" ng-pattern="/^[^\n\r]*$/" rows="3"></textarea>
```

Comment:
if this can't have multiple lines, why not just make it an `input[type="text"]`?

Reply:
remap text gets long and i want the word wrapability of a TA.

Reply:
Yeah, I suppose that makes sense. You _could_ make the number of rows dynamic so that it's always exactly big enough to contain its contents. But I won't make you do that to fix a bug.

Reply:
textareas are stretchable so i suppose if the content goes over the 3 rows, they can stretch the TA

Reply:
I think because it's easier to read long inputs with text-wrapping.